File: jysmol_python/input.py
import sys

from jysmol import Jysmol

def run():
    input = sys.argv[1]
    print('input: ', input)
    parsed = Jysmol.parse(input)
    print('parsed: ', parsed)
    stringified = Jysmol.stringify(parsed)
    print('stringified: ', stringified)
run()

File: jysmol_lua/src/input.lua
local Jysmol = require"src.jysmol"

local function run ()
   local input = arg[1]
   print("input: " .. input)
   local parsed = Jysmol.parse(input)
   -- print("parsed: " .. parsed)
   local stringified = Jysmol.stringify(parsed)
   print("stringified: " .. stringified)
end

run()
